dev-docs/bidders/brainx.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: bidder
+title: brainx
+description: Prebid brainx Bidder Adapter
+biddercode: brainx
+aliasCode: fileContainingPBJSAdapterCodeIfDifferentThenBidderCode
+tcfeu_supported: false
+dsa_supported: false
+gvl_id: none
+usp_supported: false
+coppa_supported: false
+gpp_sids: None
+schain_supported: true
+dchain_supported: false
+userId: (list of supported vendors)
+media_types: banner
+safeframes_ok: false
+deals_supported: true
+floors_supported: true
+fpd_supported: false
+pbjs: true
+pbs: false
+prebid_member: false
+multiformat_supported: will-bid-on-any
+ortb_blocking_supported: true
+sidebarType: 1
+---
+
+### Registration
+
+If you have any questions regarding set up, please reach out to your account manager or <hugh.qu@tec-do.com>.
```

Comment:
Contact must be a group alias, not an individual. Our experience is that people move around a lot in ad tech.

Reply:
The email address has been changed to the official email address